api/routes/articles.js

```diff
@@ -8,179 +8,208 @@ const pool = new Pool({
   ssl: true
 });
 
+const ArticleModel = require('../models/ArticleModel')
+
 //  GET /articles
-router.get('/',async (req, res) => {
-  const queryObject = url.parse(req.url,true).query;
```

Comment:
You do not need the `url` dependency. Read the ExpressJS documentation: https://expressjs.com/en/api.html#req.query